mobile/package.json

```diff
@@ -18,6 +18,7 @@
     "expo-status-bar": "~1.1.0",
     "expo-web-browser": "~10.0.3",
     "firebase": "9.5.0",
+    "fsevents": "^2.3.2",
```

Comment:
This is already in optionalDependencies and isn't used at runtime, can you revert it?